src/Tribe/Log/Service_Provider.php

```diff
@@ -151,7 +151,7 @@ public function add_logging_engine( array $logging_engines = [] ) {
 	/**
 	 * Enables logging in the service provider, if not already enabled.
 	 *
-	 * @since TBD
+	 * @since 4.12.15
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: c5bce384583a3ba28d7fc42021a6d36a</sub>